src/Models.ts

```diff
@@ -94,6 +97,13 @@ export interface JWSTransactionDecodedPayload {
   webOrderLineItemId: string
 }
 
+// https://developer.apple.com/documentation/appstoreserverapi/offerdiscounttype
+export enum OfferDiscountType {
+  FreeTrail = "FREE_TRIAL",
```

Comment:
There is a typo here.
```suggestion
  FreeTrial = "FREE_TRIAL",
```